pkg/runctl/controller.go

```diff
@@ -769,13 +812,30 @@ func (c *Controller) addToWorkqueueFromAssociated(obj interface{}) {
 			utilruntime.HandleError(fmt.Errorf("error decoding object tombstone, invalid type"))
 			return
 		}
-		klog.V(3).Infof("Recovered deleted object '%s' from tombstone", object.GetName())
+		klog.V(3).InfoS("Recovered deleted object from unknown state",
+			"recoveredObject", object.GetNamespace()+"/"+object.GetName())
```

Comment:
may be store namespace and name as single values.
"recoveredObjectNamespace", object.GetNamespace(),
"recoveredObjectName", object.GetName()

Reply:
I would rather prefer [`klog.KObj(object)`](https://pkg.go.dev/k8s.io/klog/v2@v2.80.1#KObj). This function will log in `namespace/namespaced-object-name` format for namespaced object or simply object name for cluster-scoped objects, [see the example](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#structured-logging-in-kubernetes).

the log line would look like - 

```
"Recovered deleted object from unknown state" recoveredObject="foo-ns/foo"
```

Reply:
Done adfbc2abfdf982d656d0b4de583541d07ca5e8a2